django/santropolFeast/member/models.py

```diff
@@ -36,11 +36,11 @@ class Meta:
     gender = models.CharField(
         max_length=1,
         choices=(
-            ('H', 'Homme'),
-            ('F', 'Femme'),
-            ('U', 'Inconnu'),
+            ('M', 'Male'),
+            ('F', 'Female'),
+            ('U', 'Unknown'),
         ),
-        default=1
+        default=2
```

Comment:
The django doc said that we must call a function that returns the default value.

ref. https://docs.djangoproject.com/en/1.9/ref/models/fields/#django.db.models.CharField

Under the section default.